src/frontend/sharedComponents/HomeHeader.tsx

```diff
@@ -25,7 +25,11 @@ export const HomeHeader: FC<BottomTabHeaderProps> = ({navigation}) => {
         <SyncIconCircle />
       </IconButton>
       <GPSPill navigation={navigation} />
-      <IconButton onPress={() => {}} testID="observationListButton">
+      <IconButton
+        onPress={() => {
+          navigation.navigate('Settings');
+        }}
+        testID="observationListButton">
```

Comment:
I updated this a temporary button to show the setting screen (otherwise this would be impossible to test)